src/Microsoft.AspNet.Razor.Runtime/TagHelpers/OrderAttribute.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
+{
+    /// <summary>
+    /// Used to override <see cref="ITagHelper"/>'s default execution order.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false, Inherited = false)]
+    public class OrderAttribute : Attribute
```

Comment:
Should this be named differently? OrderAttribute seems too generic a name - `TagHelperExecutionOrderAttribute` maybe?